README.md

```diff
@@ -0,0 +1,438 @@
+
+##### Dependencies
+
+I'll assume you're installing podofo and any required libraries into $HOME/libs . Adjust to taste, but keep it consistent.
+
+The easiest way to get any required libraries is to use [MacPorts](http://macports.org/) to install the libjpeg, fontconfig, and freetype libraries. Once you've installed MacPorts, just run:
+
+/opt/local/bin/port install fontconfig freetype jpeg tiff lua
+
+MacPorts will automatically build the libraries and their dependencies, installing them in /opt/local.
+
+If you want to distribute the libraries with your application, all you'll need to do is use install\_name\_tool to set appropriate relative paths for their linkage and include them in your application bundle - just like you do with any other libraries.
+
+##### PoDoFo its self
+
+You should be able to configure and install podofo on Mac OS X using:
+
+ cmake -G "Unix Makefiles" \\
+  -DWANT\_FONTCONFIG:BOOL=TRUE \\
+  -DCMAKE\_INSTALL\_PREFIX=/opt/podofo \\
+  -DCMAKE\_INCLUDE\_PATH=/opt/local/include \\
+  -DCMAKE\_LIBRARY\_PATH=/opt/local/lib \\
+  ../podofo
+ make
+ sudo mkdir /opt/podofo
+ sudo chown $USER /opt/podofo
+ make install
+
+change "Unix Makefiles" to "Xcode" if you want to build an XCode project instead, then instead of running \`make' and \`make install' just open the project file and work as normal
+
+#### Homebrew builds on Mac OS X
+
+For Homebrew (package manager for Mac OS X) it is quite simple to install podofo. Freetype2 and zlib should be installed on Mac OS X by default.
+
+brew install fontconfig
+brew install libjpeg
+brew install libtiff
+cd ((directory of podofo))
+cmake -G "Unix Makefiles"
+make
+make install
+
+#### CMake builds on Windows
+
+##### Windows Dependencies
+
+PoDoFo requires a couple of other open source libraries to work. At minimum it needs zlib and libjpeg. Additional functionality is enabled if libtiff and/or openssl are available.
+
+###### zlib
+
+Both MinGW and Visual Studio users should use the standard precompiled zlib dll from [zlib.org](http://zlib.org). There is no need to compile your own.
+
+Remember that you must ship `zlib1.dll` with your application. During testing it must be in the same directory as the PoDoFo-using executable or the directory containing it must be on your system PATH.
+
+###### libjpeg
+
+MinGW users can download the latest libjpeg installer from [GnuWin32](http://gnuwin32.sf.net)
+
+.
+
+For Visual Studio you will need to build libjpeg with your version of Visual Studio. Download the libjpeg sources from [http://www.ijg.org/](http://www.ijg.org/) and unpack them into a working directory. I'll assume you've used `C:\Developer\jpeg`. Once the sources are unpacked, start a visual studio command prompt and cd into `c:\developer\jpeg` then run `copy jconfig.vc jconfig.h` then `nmake /f makefile.vc /a`
+
+.
```

Comment:
Both of these look like the full stop/period should be part of the preceding paragraph.